src/main/java/org/takes/rq/RqWithDefaultHeader.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015 Yegor Bugayenko
```

Comment:
@dmzaytsev we have similar ticket in Netbout ( https://github.com/yegor256/netbout/issues/909 ) and it's postponed because it qulice issue: https://github.com/teamed/qulice/issues/411
So what should I do here? 2016 and failing build or 2015 and green build?